config/routes.rb

```diff
@@ -21,6 +21,12 @@
 
     resources :locations, only: [:index, :show]
 
+    resources :appointment_summaries, only: %i(new create), path: 'summary-document' do
+      post :view, on: :collection
```

Comment:
You could nest these three `post` routes inside a `collection` block instead, then we can drop the `on` arguments for all three.